tests/acceptance/features/apiGraph/createUser.feature

```diff
@@ -0,0 +1,42 @@
+@api @skipOnOcV10
+Feature: create user
+  Only user with admin permissions can create new user
+
+  Note - this feature is run in CI with ACCOUNTS_HASH_DIFFICULTY set to the default for production
+  See https://github.com/owncloud/ocis/issues/1542 and https://github.com/owncloud/ocis/pull/839
+
+  Background:
+    Given user "Alice" has been created with default attributes and without skeleton files
+
+  Scenario Outline: the admin creates a user
+    Given the administrator has given "Alice" the role "Admin" using the settings api
+    When the user "Alice" creates new user using GraphApi with settings:
+      | userName    | <userName>    |
+      | displayName | <displayName> |
+      | email       | <email>       |
+      | password    | <password>    |
+    Then the HTTP status code should be "<code>"
+    Examples:
```

Comment:
```
And the user "" should exist
```
imo, the HTTP status code can be right sometimes and the user may not get created. a then step like this will make sure the created user is fine and there in the server.